model/docs/pyrenew_demo.qmd

```diff
@@ -130,6 +139,8 @@ hospmodel = HospitalizationsModel(
     latent_infections=latent_infections,
     Rt_process=Rt_process
     )
+# Damon: I don't really get why there is a hospitalizations model as a concept.
+# Damon: Maybe the scope of the project is so limited that it makes sense, but one can easily imagine additional data sources (Wastewater, separate hosp and ICU admissions). Would each of those get a separate class?
```

Comment:
Indeed, that's the whole point. The current state features a handful of models and classes, but needs to be extended. Most of the required features (including other data models) are listed under the repo's issues.